app/controllers/students_controller.rb

```diff
@@ -23,7 +23,7 @@ def new
     @student.past_education_records.new
     @student.point_of_contacts.new
     @student.internship_records.new
-    # @student.remarks.new
+    @student.remarks.new
```

Comment:
i was thinking of not showing any remark at all.

Reply:
So just the add button under the Remarks title?

Reply:
Yeah. Isn't it already there?